src/baseindicator.js

```diff
@@ -41,7 +41,7 @@ const Me = ExtensionUtils.getCurrentExtension();
 const Convenience = Me.imports.src.convenience;
 const SETTINGS_ID = 'org.gnome.shell.extensions.cpupower';
 
-const CPUFreqBaseIndicator = new Lang.Class({
+var CPUFreqBaseIndicator = new Lang.Class({
```

Comment:
Can you give a hint why you changed all `const` to `var`?

Reply:
Yes, this is showing up in my logs:
```
Oct 17 06:32:18 holodeck.local gnome-shell[1141]: Some code accessed the property 'Indicator' on the module 'indicator'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols t
o be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
```